classes/autoptimizeStyles.php

```diff
@@ -351,7 +354,7 @@ public function read( $options )
                         $new_tag = $this->optionally_defer_excluded( $new_tag, $url );
```

Comment:
OK, next; this line will have to move inside a `if ( '' !== $new_tag)` to avoid we add stuff to an empty $new_tag when `defer` is on.